data/json/obsolete.json

```diff
@@ -1488,5 +1640,26 @@
       [ "VISE", 1 ]
     ],
     "use_action": [ "HAMMER", "CROWBAR" ]
+  },
+  {
+    "id": "towel_soiled",
+    "type": "TOOL_ARMOR",
+    "repairs_like": "towel",
+    "symbol": ",",
+    "color": "magenta",
+    "name": { "str": "soiled towel" },
+    "description": "This is a large towel, covered in a thick, gooey liquid.  You'll need to wash it in some water to make it usable again.  Don't panic.",
+    "price": 2000,
+    "price_postapoc": 0,
+    "material": [ "cotton" ],
+    "covers": [ "torso", "leg_l", "leg_r" ],
+    "weight": "400 g",
+    "volume": "500 ml",
+    "to_hit": -1,
+    "revert_to": "towel",
+    "use_action": [ "TOWEL" ],
+    "flags": [ "WET" ],
+    "coverage": 50,
+    "material_thickness": 1
```

Comment:
I suggest adding a migration entry to `json/items/migration.json` instead of obsoleting the old item id.
```
  {
    "id": "towel_soiled",
    "type": "MIGRATION",
    "replace": "towel",
    "flags": [ "FILTHY" ]
  }
```

Reply:
thank you, i ll update it